config/src/vespa/config/frt/frtsource.cpp

```diff
@@ -76,7 +76,7 @@ FRTSource::getConfig()
 void
 FRTSource::erase(FRT_RPCRequest * request) {
     std::lock_guard guard(_lock);
-    _inflight.erase(request);
+    assert(1 == _inflight.erase(request));
```

Comment:
you should never put actual code inside an assert

Reply:
Ouch..